tests/e2e/matmul/CMakeLists.txt

```diff
@@ -50,6 +75,33 @@ iree_generated_trace_runner_test(
     "--iree-opt-data-tiling=false"
   LABELS
 
+  TARGET_CPU_FEATURES_VARIANTS
+    "default"
+    "arm_64:sve:+sve"
+)
```

Comment:
Looks like this check is working as intended to exclude tests from certain platforms: https://github.com/openxla/iree/blob/39afb04919fa5205ed074e8dceb1b727449038a8/build_tools/cmake/iree_trace_runner_test.cmake#L341-L346

Windows and macOS builds are passing.